tasks/task_16/provider/google_provider/google_provider_helper.go

```diff
@@ -0,0 +1,86 @@
+package google_provider
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"golang.org/x/oauth2"
+)
+
+func getClient(ctx context.Context, config *oauth2.Config, tokFile string) (*http.Client, error) {
```

Comment:
all this is incorrect, please use machine auth to avoid browser opening etc. tokenFile file is also useless